node/actors/executor/src/lib.rs

```diff
@@ -36,6 +42,14 @@ impl fmt::Debug for Validator {
     }
 }
 
+impl fmt::Debug for Attester {
+    fn fmt(&self, fmt: &mut fmt::Formatter<'_>) -> fmt::Result {
+        fmt.debug_struct("AttesterExecutor")
+            .field("key", &self.key)
```

Comment:
this can be just derived. Also "ValidatorExecutor" should be just "Validator" - we forgot to change that.